Untitled Game/src/Actors/LirikYaki.gd

```diff
@@ -57,32 +66,37 @@ func evaluatePlayerInput() -> Vector2:
 		Input.get_action_strength("move_right") - Input.get_action_strength("move_left"),
 		Input.get_action_strength("move_down") - Input.get_action_strength("move_up")
 	)
-	if(direction != Vector2.ZERO):
-		_directionFacing = direction
```

Comment:
You can't keep _directionFacing state if we are no longer going to keep updating it with the direction.  This is used in damage direction calculations.

Reply:
Fixed this in latest commit 9917d3fc3008aaa25ad3eb7ba8ba27cec4a1f1a2.